@navikt/core/css/darkside/button.darkside.css

```diff
@@ -273,6 +277,10 @@
   stroke: rgb(255 255 255 / 0.3);
 }
 
+.navds-button--loading > :is(:not(.navds-loader)) {
```

Comment:
Why use `:is` here?

Reply:
Den hang igjen fra tidligere. Fjernet både her og i button.css